extensions/3DTILES_implicit_tiling/README.md

```diff
@@ -0,0 +1,963 @@
+                    0, 0, 5
+                ]
+            },
+            "refine": "REPLACE",
+            "tilingScheme": "octree",
+            "contentExtension": "glb",
+            "tilesetExtension": "json",
+            "content": {
+                "levelOffset": 4,
+                "levelOffsetFill": 1
+            },
+            "subdivision": {
+                "completeLevels": 3
+            }
+        }
+    }
+}
+```
+
+##### Directory Structure
+
+```
+.
+└── tileset.json/
+    ├── L0/
+    │   └── 0/0/0/
+    │       └── 0/0/0.glb
+    ├── L1/
+    │   ├── 0/0/0/
+    │   │   └── 0/0/0.glb
+    │   ├── 0/1/0/
+    │   │   └── 0/1/0.glb
+    │   ├── 1/0/0/
+    │   │   └── 1/0/0.glb
+    │   ├── 1/1/0/
+    │   │   └── 1/1/0.glb
+    │   ├── 0/0/1/
+    │   │   └── 0/0/1.glb
+    │   ├── 0/1/1/
+    │   │   └── 0/1/1.glb
+    │   ├── 1/0/1/
+    │   │   └── 1/0/1.glb
+    │   └── 1/1/1/
+    │       └── 1/1/1.glb
+    ├── L2/
+    │   └── ...
+    └── ...    
+```
+
+#### Double Headed Quadtree (as 2 separate tilesets)
+
+##### Root 0 Tileset - tileset.json
+
+```json
+{
+    "asset": {
+        "version": "2.0.0-alpha.0"
+    },
+    "geometricError": 500,
+    "extensions": {
+        "3DTILES_implicit_tiling": {
+            "boundingVolume": {
+                "box": [
+                    0, 0, 0,
+                    5, 0, 0,
+                    0, 5, 0,
+                    0, 0, 5
+                ]
+            },
+            "tilingScheme": "quadtree",
+            "refine": "REPLACE",
+            "contentExtension": "glb",
+            "tilesetExtension": "json",
+            "subdivision": {
+                "bufferView": 0
+            },
+            "content": {
+                "bufferView": 1
+            }
+        }
+    }
+}
+```
+
+##### Root 1 Tileset - tileset.json
+
+```json
+{
+    "asset": {
+        "version": "2.0.0-alpha.0"
+    },
+    "geometricError": 500,
+    "extensions": {
+        "3DTILES_implicit_tiling": {
+            "boundingVolume": {
+                "box": [
+                    10, 0, 0,
+                    5, 0, 0,
+                    0, 5, 0,
+                    0, 0, 5
+                ]
+            },
+            "tilingScheme": "quadtree",
+            "refine": "REPLACE",
+            "contentExtension": "glb",
+            "tilesetExtension": "json",
+            "subdivision": {
+                "bufferView": 0
+            },
+            "content": {
+                "bufferView": 1
+            }
+        }
+    }
+}
+```
+
+##### Directory Structure
+
+```
+.
+├── Tileset 0/
+│   └── tileset.json/
+│       ├── L0/
+│       │   └── 0/0/
+│       │       └── 0/0.glb
+│       ├── L1/
+│       │   ├── 0/0/
+│       │   │   └── 0/0.glb
+│       │   ├── 0/1/
+│       │   │   └── 0/1.glb
+│       │   ├── 1/0/
+│       │   │   └── 1/0.glb
+│       │   └── 1/1/
+│       │       └── 1/1.glb
+│       ├── L2/
+│       │   └── ...
+│       └── ...
```

Comment:
```
└── tileset.json/
    ├── 0/
    │   └── 0/
    │       └── 0.glb
    ├── 1/
    │   ├── 0/
    │   │   └── 0.glb
    │   │   └── 1.glb
    │   ├── 1/
    │   │   └── 0.glb
    │   │   └── 1.glb
    ├── 2/
    │   └── ...
    └── ...    
```

Feels like a more natural way to express this

Reply:
Updated directory structure in examples.